src/config_parser.ml

```diff
@@ -251,7 +252,8 @@ let dhcp_host end_of_directive =
                in
                match Ohex.decode hex_name with
                | name -> return (`Client_id name)
-               | exception Invalid_argument _ -> fail "bad hex constant"
+               | exception Invalid_argument _ ->
+                   fail (Fmt.str "bad hex constant: %S" name)
```

Comment:
here, is name bound? should the value x of `Invalid_argument x` be present in the error?

Reply:
Yes, `name` is `hex_name` but with the colons preserved. I can add the error from ohex to the message.

Reply:
I pushed a commit that adds the error message. They now look like the following:
```
Parse error in "00:20:11:22:33:44,id:42,192.168.0.99,00:20:00:11:22:33,set:test,id:a:bacon,hostname": : bad hex constant: bad character: "a:bacon"
Parse error in "00:20:11:22:33:44,id:42,192.168.0.99,00:20:00:11:22:33,set:test,id:be:f,hostname": : bad hex constant: leftover byte in hex string: "be:f"
```